packages/core/src/components/menu/menuItem.tsx

```diff
@@ -25,12 +25,11 @@ export interface IMenuItemProps extends IActionProps, ILinkProps {
      */
     label?: string | JSX.Element;
 
-    /** Props to spread to `Popover`. Note that `content` cannot be changed. */
-    popoverProps?: Partial<IPopoverProps> & object;
+    /** Props to spread to `Popover`. Note that `content` and `minimal` cannot be changed. */
+    popoverProps?: Partial<IPopoverProps>;
```

Comment:
`& object` was there intentionally... IIRC without it, you can assign literally anything non-null to this field.

Reply:
`& object` messes up the docs, which i take to be more important. a prop called `*Props` is expected to be an object.
![screen shot 2018-01-31 at 3 06 45 pm](https://user-images.githubusercontent.com/464822/35652424-6ac17148-0698-11e8-9632-91061c85be96.png)